src/overlays/actors/ovl_En_Invadepoh/z_en_invadepoh.c

```diff
@@ -2617,7 +2617,7 @@ void func_80B48848(EnInvadepoh* this, GlobalContext* globalCtx) {
         Math_StepToS(&this->behaviorInfo.unk4C, 0x7D0, 0x46);
     }
     func_80B43E6C(this, 6, this->behaviorInfo.unk4C, 0x46);
-    if (((this->actor.flags & 0x40) == 0x40) &&
+    if (((this->actor.flags & ACTOR_FLAG_40) == ACTOR_FLAG_40) &&
```

Comment:
This is the macro `CHECK_FLAG_ALL`. Since you are updating the actor flags could you change every instance of this pattern on actor flags to use the macro?

```suggestion
    if (CHECK_FLAG_ALL(this->actor.flags, ACTOR_FLAG_40) &&
```